### ISSUE TITLE
Cajon >Parralax always present

### ISSUE THREAD

**Issue by gtbu** [Typesetter/Typesetter#679]
I had a look at the source of my RC%" - installations  and saw that all had
<link rel="stylesheet" type="text/css" href="/themes/Cajon_Scss/addons/CajonParallax/CajonParallax.css" />
....
<script type="text/javascript" src="/themes/Cajon_Scss/addons/CajonParallax/CajonParallax.js"></script>
<script type="text/javascript" src="/themes/Cajon_Scss/addons/CajonParallax/jquery.scrollspeed/jQuery.scrollSpeed.js"></script>

though Cajon theme is only available and not used. This may influence other themes and puts a load on the client- so my only momentary solution is to delete it -if not used .....

**Comment by juek** [comment]
True. I already improved current page -> theme detection in the core but not using it in Cajón.
In fact, scrollspeed.js is somewhat outdated since chrome/chromium based browsers do a better job in smooth scrolling for some versions now. Not even close to Firefox but, well, better than before.

For the Cajón Parallax section type, well it **can** be used in other themes too. Cajón Parallax does not depend on the theme. The problem when only allowing a section type (and its editor components) in a particular theme is that it would cause an error when switching (a page or the whole site) from the allowed theme to a different one. So the question is if it's worth the hassle to block it. 

IMO there is no good solution for that, other than **not** binding section types and other functionality to themes. E.g. when section types are made only for a distinct Bootstrap version, we get even more problems. On the other hand, it makes sense to do it to use Bootstrap as efficiently as possible.

But maybe it's less of an issue. In the end. switching between themes or using multiple ones in parallel is rarely the case in production state of a site.

**Comment by gtbu** [comment]
I would personally rewrite the template....
(P.S. and also scssphp 1.4 or the [newest master](https://github.com/scssphp/scssphp) )

But scssphp v1.3+ has a problem with the bootstrap4-template: rem...px 

This sass-problem has to do with _variables.scss -also in [other](https://laracasts.com/discuss/channels/laravel/error-in-installing-bootstrap-400) cmsses (one '[solution](https://davidwalsh.name/rem-px-browser-function-sass)') ....or [sass-rem](https://www.npmjs.com/package/sass-rem).. or [Git](https://github.com/pierreburel/sass-rem)

**Comment by gtbu** [comment]
Is Typesetter dead ? Grafik Krausz is offline !

**Comment by mahotilo** [comment]
> Grafik Krausz is offline !

Have you just noticed this? It happened 3 or more months ago.

But @juek has posted here since then, so I hope he is OK and that all the changes are only for the better.
